### PR TITLE
OSSM-3067 [DOC]  Update security for jwksResolverCA

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-3067](https://issues.redhat.com//browse/OSSM-3067) [DOC]  Update security for jwksResolverCA

Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OSSM-3067

Link to docs preview:
https://68087--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-security#ossm-configure-JWKS-resolver-CA_ossm-security

QE review:
- [x] QE has approved this change.

Additional information:
Only adds new module for jwksResolverCA

### CODE REVIEW

**Review by yxun — APPROVED** [review]

**Review by pbajjuri20 — APPROVED** [review]
LGTM..!

**Review by cbippley — COMMENTED** [review]
LGTM!

File: modules/ossm-configuring-jwks-resolver-ca.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+.Procedure
+
+. Edit the `ServiceMeshControlPlane` spec file:
++
+[source, yaml]
+----
+$ oc edit smcp <smcp-name>
+----
+
+. Enable `mtls` for the data plane by setting the value of the `mtls` field to true in the `ServiceMeshControlPlane` spec, as shown in the following example:
```
Comment (mburke5678) [inline modules/ossm-configuring-jwks-resolver-ca.adoc:20]:
```suggestion
. Enable `mtls` for the data plane by setting the value of the `mtls` field to `true` in the `ServiceMeshControlPlane` spec, as shown in the following example:
```

File: modules/ossm-configuring-jwks-resolver-ca.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        -----BEGIN CERTIFICATE-----
+        [...]
+        [...]
+        -----END CERTIFICATE-----
+...
+----
+
+. Save the changes. {product-title} automatically applies them.
+
+. A `ConfigMap` such as `pilot-jwks-cacerts-<SMCP name>` is created with the CA `.pem data`.
```
Comment (mburke5678) [inline modules/ossm-configuring-jwks-resolver-ca.adoc:40]: I am assuming that this is not a step the user needs to perform, but a system response?
```suggestion
+
A `ConfigMap`, such as `pilot-jwks-cacerts-<SMCP name>`, is created with the CA `.pem data`.
```

**Comment by gwynnemonahan** [comment]
@mburke5678 Changes made.

**Pull request merged**